install.sh

```diff
@@ -78,20 +92,21 @@ https://hosts-file.net/ad_servers.txt
 https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext
 EOF
                 echo "[i] File created."
+                chmod u=rwx,g=rw,o=rw "hostssources.lst" # Allows the user to edit the file which is created by root.
 
                 # add cronjob
                 read -r -p "[?] Create a cronjob/systemd-service which updates /etc/hosts with new adservers once a week? [systemd/cronjob/N] " REPLY
                 case "${REPLY}" in
                     "cronjob" | "Cronjob" | "CRONJOB" | "CronJob" | "crontab" | "Crontab" | "CRONTAB" | "CronTab" | "cron" | "Cron" | "CRON" | "c" | "C")
                         echo "[i] Creating cronjob..."
                         line="1 12 */5 * * ${PWD}/adaway-linux.sh"
-                        (sudo crontab -u root -l; echo "$line" ) | sudo crontab -u root -
+                        (crontab -u root -l; echo "$line" ) | crontab -u root -
                         ;;
                     "systemd" | "Systemd" | "SYSTEMD" | "sys" | "Sys" | "SYS" | "S" | "s")
                         echo "[i] Creating systemd service..."
 
                         # create .service file
-                        sudo sh -c "cat > \"${SYSTEMD_DIR}/adaway-linux.service\" <<EOL
+                        sh -c "cat > \"${SYSTEMD_DIR}/adaway-linux.service\" <<EOL
```

Comment:
Now we can use a simple `cat > ... <<EOL`. I added the `sh` thing, because piping `stdout` into this file needs root permissions. If we already have root permissions, we don't need this ugly workaround. 😉